charts/stable/immich/values.yaml

```diff
@@ -168,6 +168,16 @@ persistence:
         main: {}
       microservices:
         microservices: {}
+  externallibrary:
+    enabled: true
+    mountPath: /mnt/media
+    readOnly: true
```

Comment:
- We do not recommend using `/mnt` inside the container, unless its absolutely needed. Is that the case?
- readOnly will get overwritten from what's on the UI, which by default is false

Reply:
- no, not needed. I can change the mount path. What would be your recommendation? e.g.  `/media`?
- ok, good to know.

Reply:
- yes
- lets remove it then, in order to not confuse anyone